Find a functionally equivalent code:com.android.tools.layoutlib.create.DelegateClassAdapterTest.ClassLoader2.callGet_Original:COMMENT
Method Modifier: public      
Comment:/**
 * Accesses the "_Original" methods for {@link OuterClass#get}
 * or {@link InnerClass#get}via reflection.
 */

@@ -1,5 +1,5 @@
 {
-    Method m = instance.getClass().getMethod("get_Original", new Class<?>[] { int.class, long.class });
-    Object result = m.invoke(instance, new Object[] { a, b });
-    return ((Integer) result).intValue();
+    Method m = instance.getClass().getMethod("get_Original", int.class, long.class);
+    Object result = m.invoke(instance, a, b);
+    return (Integer) result;
 }

